vllm/model_executor/layers/rotary_embedding.py

```diff
@@ -610,6 +610,16 @@ def forward(
         return query.flatten(-2), key.flatten(-2)
 
 
+class GemmaRotaryEmbedding(RotaryEmbedding):
```

Comment:
similar comment as before -- should we name this `Gemma2RotaryEmbedding`?

Reply:
Good point. Actually Gemma1 also has to use GemmaRotaryEmbedding instead of RotaryEmbedding. This will be fixed in https://github.com/vllm-project/vllm/pull/5913